### DISCUSSION TITLE
Running the model ERROR

### DISCUSSION THREAD

**Discussion by hadominh** [pyswmm/pyswmm#339] [Q&A] [answered]
Hi, I tried to run the model by pyswmm. however, they give me this error

ImportError: DLL load failed while importing _solver: The specified module could not be found.
How can i solve this problem? Anything that i did wrong with this ?
![image](https://user-images.githubusercontent.com/80476073/168867215-96499671-80c5-45a0-8f07-7e8bbf2ca2fb.png)

**Comment by hadominh** [comment]
![image](https://user-images.githubusercontent.com/80476073/168867520-388877cc-e28e-498a-9417-7e6149ccd9ea.png)
my test is very simple like in the picture

**Comment by jennwuu** [comment, marked as answer]
Hey @hadominh thanks for checking out pyswmm!

Can you share the following information about your development environment?
- pyswmm package
- swmm-python package
- Python version (32 bit vs 64 bit)

We know there are issues when using Anaconda currently, but plan to address in the next release. Can you use regular Python in the meantime?

**Reply by hadominh** [reply]
Hi jennwuu, 
I use Python 3.9.7 64bit.
pyswmm package version 1.1.1
swmm-python package that i dont know. I just only install pyswmm by pip install pyswmm. 
If it is a package that i need to install - then i dont find the instruction for this. 
Could you please give me the instruction for swmm-python package? 
Thank you!

**Reply by jennwuu** [reply]
This is an known issue when using Anaconda. This should be addressed in the next release.

Please try the answer from this discussion:
https://github.com/OpenWaterAnalytics/pyswmm/discussions/323

**Reply by hadominh** [reply]
Woah! 
That is great! Seem it is working - thank you !

**Reply by jennwuu** [reply]
No problem!
